src/taipy/core/taipy.py

```diff
@@ -64,6 +64,20 @@ def set(entity: Union[DataNode, Task, Pipeline, Scenario, Cycle]):
         return _DataManagerFactory._build_manager()._set(entity)
 
 
+def is_submittable(entity: Union[Scenario, ScenarioId, Pipeline, PipelineId, Task, TaskId]) -> bool:
+    """Indicate if an entity can be submitted.
+    Returns:
```

Comment:
```suggestion
    """Indicate if an entity can be submitted.

    Returns:
```
I think in the convention, there should be an empty line here.